package.json

```diff
@@ -6,12 +6,11 @@
     "dev": "cd app && npm run dev",
     "build": "cd app && npm run build",
     "test": "tsc -b . && cd app && npm test",
-    "postinstall": "npm run -s install:app && npm run -s install:bin",
+    "postinstall": "npm run -s install:app && npm run -s install:bin && npm run -s install:csaf-validator-lib",
```

Comment:
This is just listed in "postinstall". Does that automatically cover "dev" and "build"?

Reply:
Yes, since "dev" and "build" imply `npm install`